openwisp_radius/api/permissions.py

```diff
@@ -5,10 +5,14 @@
 from rest_framework.exceptions import PermissionDenied
 from rest_framework.permissions import BasePermission
 
+from ..settings import REGISTRATION_API_ENABLED
```

Comment:
use `from . import settings as app_settings` for consistency, use `app_settings.REGISTRATION_API_ENABLED` in the code, which also makes it easier to mock/change the value during tests.